charts/hivemq-operator/values.yaml

```
@@ -2,7 +2,8 @@ global:
   rbac:
     create: true
     # Create a PodSecurityPolicy, cluster role, role binding and service account for the HiveMQ pods and assign the service account to them.
-    pspEnabled: true
+    # Disable now by default as it was completely deprecated in K8s v1.25 https://kubernetes.io/docs/concepts/security/pod-security-policy/
```

Nitpick, here. It was deprecated in v1.21, and removed in v1.25

```suggestion
    # Disable now by default as it was completely removed in K8s v1.25 https://kubernetes.io/docs/concepts/security/pod-security-policy/
```